validator/project_validator.go

```diff
@@ -1799,8 +1826,8 @@ func checkTaskGroups(p *model.Project) ValidationErrors {
 		// validate that teardown group timeout is not over MaxTeardownGroupTimeout
 		if tg.TeardownGroupTimeoutSecs > int(globals.MaxTeardownGroupTimeout.Seconds()) {
 			errs = append(errs, ValidationError{
-				Message: fmt.Sprintf("task group '%s' has a teardown task timeout of %d seconds, which exceeds the maximum of %d seconds", tg.Name, tg.TeardownGroupTimeoutSecs, int(globals.MaxTeardownGroupTimeout.Seconds())),
 				Level:   Warning,
+				Message: fmt.Sprintf("task group '%s' has a teardown task timeout of %d seconds, which exceeds the maximum of %d seconds", tg.Name, tg.TeardownGroupTimeoutSecs, int(globals.MaxTeardownGroupTimeout.Seconds())),
```

Comment:
This is just swapping `Level` and `Message`. We don't need to do this but I did it when I was manually verifying during testing.